client/src/components/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -15,12 +16,12 @@ interface Props {
 const ProfilePhoto = ({ loggedInUser }: Props): JSX.Element => {
   const classes = useStyles();
 
-  const [image, setImage] = useState<any>({});
-
   const handleUpload = (event: ChangeEvent<HTMLInputElement>) => {
-    event.preventDefault();
-    const uploadedImage = event.target.files;
-    setImage(uploadedImage);
+    const image = event.target.files![0];
+
+    const avatar = image;
```

Comment:
are you getting the correct file here? (you can confirm by console logging `image` here)

If so, then by saving it in a variable, you're likely losing the data as you proceed calling `uploadAvatar`. 

Let's assume you are getting the right form data here :) In which we have to continue 

Try this out! https://stackoverflow.com/a/64493177/14204824 Key is to create a new `FormData` and pass it to the backend letting it know that you're attaching something along the way

https://stackoverflow.com/a/50657625/14204824 may be helpful. 

```
fetch("ProposalData/UploadFile", {
            method: 'POST',
            credentials: 'include',
            headers: {
                'Accept': 'application/json, */*',
            },
            body: formData
        })
```

Reply:
it does capture the image per console.log, and I attempted to set it to formData, but I still get the same error. I tried adding the 'Accept' option under headers, but the fetchOptions page didn't like it, so I tried to alter it to add it, but it still doesn't work correctly.